libraries/push/impl/src/main/kotlin/io/element/android/libraries/push/impl/notifications/conversations/DefaultNotificationConversationService.kt

```diff
@@ -49,15 +50,17 @@ class DefaultNotificationConversationService @Inject constructor(
             if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N_MR1) ShortcutInfo.SHORTCUT_CATEGORY_CONVERSATION else null
         )
 
-        val client = matrixClientProvider.getOrRestore(sessionId).getOrThrow()
+        val client = matrixClientProvider.getOrRestore(sessionId).getOrNull() ?: return
         val imageLoader = imageLoaderHolder.get(client)
+
+        val defaultShortcutIconSize = ShortcutManagerCompat.getIconMaxWidth(context)
+        val useDarkTheme = context.resources.configuration.uiMode and Configuration.UI_MODE_NIGHT_MASK == Configuration.UI_MODE_NIGHT_YES
         val icon = bitmapLoader.getRoomBitmap(roomAvatarUrl, imageLoader)?.let(IconCompat::createWithBitmap)
-            ?: InitialsAvatarBitmapGenerator()
-                .generateBitmap(512, AvatarData(id = roomId.value, name = roomName, size = AvatarSize.RoomHeader))?.let(
-                IconCompat::createWithAdaptiveBitmap
-            )
+            ?: InitialsAvatarBitmapGenerator(useDarkTheme = useDarkTheme)
+                .generateBitmap(defaultShortcutIconSize, AvatarData(id = roomId.value, name = roomName, size = AvatarSize.RoomHeader))
+                ?.let(IconCompat::createWithAdaptiveBitmap)
 
-        val shortcutInfo = ShortcutInfoCompat.Builder(context, roomId.value)
+        val shortcutInfo = ShortcutInfoCompat.Builder(context, "$sessionId-$roomId")
```

Comment:
Unfortunately I only noticed this change now, this breaks the prioritization of notifications as the ids haven't been adjusted in the notifications itself. See

https://github.com/element-hq/element-x-android/blob/2fbfd25fd169c322d8d4cf376233f588bde23661/libraries/push/impl/src/main/kotlin/io/element/android/libraries/push/impl/notifications/factories/NotificationCreator.kt#L139

Reply:
This can be tested using `adb shell dumpsys notification --noredact` and looking for `shortcut=` where in `<id> found valid? <boolean>` `id` should match a registered shortcut in `adb shell cmd shortcut get-shortcuts io.element.android.x` and `boolean` should be true.